components/blocks/aboutUs.tsx

```diff
@@ -290,31 +297,39 @@ const OfficeInfo = ({ office }) => {
   );
 };
 
-const OpenStatus = ({ state }) => {
-  const stateInfo = States[state];
-  const now = dayjs().tz(stateInfo?.timeZone);
-  const isWeekend = [DAY_KEYS.Saturday, DAY_KEYS.Sunday].some(
-    (x) => x === now.day()
-  );
-  const currentHour = now.hour();
+type Status = "Open" | "Closed" | "";
 
-  let status, statusClass;
-  if (
-    isWeekend ||
-    currentHour < WORKING_TIME.Open ||
-    WORKING_TIME.Close < currentHour
-  ) {
-    statusClass = "bg-sswRed";
-    status = "Closed";
-  } else {
-    statusClass = "bg-green-400";
-    status = "Open";
-  }
+type OpenStatusProps = {
+  state?: State;
+};
+
+const OpenStatus = ({ state }: OpenStatusProps) => {
+  const [status, setStatus] = useState<Status>("");
+  const timeZone = states[state]?.timeZone;
+
+  useEffect(() => {
+    const now = dayjs.utc().tz(timeZone);
+
+    const isWeekend = [DAY_KEYS.Saturday, DAY_KEYS.Sunday].some(
+      (x) => x === now.day()
+    );
+    const currentHour = now.hour();
+
+    if (
+      isWeekend ||
+      currentHour < WORKING_TIME.Open ||
+      WORKING_TIME.Close < currentHour
```

Comment:
this should be in between the working hours so I think we should use && operator instead